ui/shared/GasSelector.qml

```diff
@@ -95,11 +95,8 @@ Item {
         id: buttonAdvanced
         anchors.verticalCenter: prioritytext.verticalCenter
         anchors.right: parent.right
-        text: advancedMode ? 
-            //% "Use suggestions"
-            qsTrId("use-suggestions") : 
-            //% "Use custom"
-            qsTrId("use-custom")
+        //% "use-suggestions"
+        text: { advancedMode ? qsTrId("use-suggestions") : qsTrId("use-custom") }
```

Comment:
That's not correct. The `qsTrId`s need to be on different lines and the `//%` need to be on top.

Reply:
The old code was right. It probably just missing the `.qm` files (what is generated by `lrelease`. So IMO, you can just revert this line and re-run `lrelease`